packages/react-components/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "react-components",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "clean": "shx rm -rf dist/",
+    "precompile": "npm run clean",
+    "compile": "tsc",
+    "watch": "tsc --watch",
```

What about renaming `watch` to `dev`? This way, the developer only needs to execute `npm run dev` in the root.

```suggestion
    "dev": "tsc --watch",
```

---

I didn't thought about it, I'll change to dev